mesh/node.c

```diff
@@ -1426,6 +1785,8 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 	struct managed_obj_request *req = user_data;
 	const char *path;
 	struct mesh_node *node = NULL;
+	struct l_queue *net_keys;
+	struct keyring_net_key *net_key;
```

Comment:
`mesh/node.c:1953:3: error: ‘net_key’ may be used uninitialized in this function [-Werror=maybe-uninitialized]`, initialize to `NULL`